generate_secrets.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cp screwdriver-api-secrets.tmpl screwdriver-api-secrets.yaml
+
+for name in '{{cookiepassword}}' '{{encryptionpassword}}' '{{hashingpassword}}'
+do
+    value=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-zA-Z0-9 | head -c 32 | base64`
```

Comment:
Does this need to be inside the loop?

Reply:
this is generating 3 random values. put it in the loop to avoid doing that 3 times

Reply:
this is generating 3 random values. put it in the loop to avoid doing that 3 times